build.psm1

```diff
@@ -1012,7 +1012,7 @@ function Start-PSPester {
     # All concatenated commands/arguments are suffixed with the delimiter (space)
```

Comment:
@JamesWTruher, your last commit had 5 failures in `PowerShell-CI-windows`
Resources strings in Microsoft.PowerShell.ConsoleHost (was -ResGen used with Start-PSBuild).'ManagedEntranceStrings' should be an available type and the strings should be correct
```powershell
Expected strings to be the same, but they were different.
Expected length: 802
Actual length:   809
Strings differ at index 19.
Expected: '...xe] [-Logi...'
But was:  '...xe] [[-Fil...'
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ResourceValidation\TestRunner.ps1: line 64
64:                     NormalizeLineEnd($resourceStringToCheck) | Should -Be (NormalizeLineEnd($inresource.value))
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -login switch for -Login and behaves correctly
```powershell
Expected $null or empty, but got @(<empty>, 'Usage: pwsh[.exe] [[-File] <filePath> [args]]', '                  [-Command { - | <script-block> [-args <arg-array>]', '                                | <string> [<CommandParameters>] } ]', '                  [-ConfigurationName <string>] [-CustomPipeName <string>]', '                  [-EncodedCommand <Base64EncodedCommand>]', '                  [-ExecutionPolicy <ExecutionPolicy>] [-InputFormat {Text | XML}]', '                  [-Interactive] [-LoadProfile] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]', '                  [-OutputFormat {Text | XML}]  [-SettingsFile <filePath>] [-STA] [-Version]', '                  [-WindowStyle <style>] [-WorkingDirectory <directoryPath>]').
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 302
302:                 $result | Should -BeNullOrEmpty
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -Login switch for -Login and behaves correctly
```powershell
Expected $null or empty, but got @(<empty>, 'Usage: pwsh[.exe] [[-File] <filePath> [args]]', '                  [-Command { - | <script-block> [-args <arg-array>]', '                                | <string> [<CommandParameters>] } ]', '                  [-ConfigurationName <string>] [-CustomPipeName <string>]', '                  [-EncodedCommand <Base64EncodedCommand>]', '                  [-ExecutionPolicy <ExecutionPolicy>] [-InputFormat {Text | XML}]', '                  [-Interactive] [-LoadProfile] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]', '                  [-OutputFormat {Text | XML}]  [-SettingsFile <filePath>] [-STA] [-Version]', '                  [-WindowStyle <style>] [-WorkingDirectory <directoryPath>]').
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 302
302:                 $result | Should -BeNullOrEmpty
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -LOGIN switch for -Login and behaves correctly
```powershell
Expected $null or empty, but got @(<empty>, 'Usage: pwsh[.exe] [[-File] <filePath> [args]]', '                  [-Command { - | <script-block> [-args <arg-array>]', '                                | <string> [<CommandParameters>] } ]', '                  [-ConfigurationName <string>] [-CustomPipeName <string>]', '                  [-EncodedCommand <Base64EncodedCommand>]', '                  [-ExecutionPolicy <ExecutionPolicy>] [-InputFormat {Text | XML}]', '                  [-Interactive] [-LoadProfile] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]', '                  [-OutputFormat {Text | XML}]  [-SettingsFile <filePath>] [-STA] [-Version]', '                  [-WindowStyle <style>] [-WorkingDirectory <directoryPath>]').
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 302
302:                 $result | Should -BeNullOrEmpty
```
ConsoleHost unit tests.-Login pwsh switch.Accepts the -log switch for -Login and behaves correctly
```powershell
Expected 0, but got 64.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 303
303:                 $LASTEXITCODE | Should -Be 0
```